src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import AceEditor from 'react-ace';
+
+import "ace-builds/src-noconflict/mode-css";
+import "ace-builds/src-noconflict/theme-github";
+
+function App() {
+
+  const [newStyle, setNewStyle] = useState();
+
+  const onChange = (newValue) => {
+    setNewStyle(newValue);
+
+    document.getElementById('style-it').innerHTML = newStyle;
+  }
+
+  //sadly it doesn't work and I can't figure out why :(
+
+  return (
+    <div className="App">
+      <style id='style-it'/>
+      <p>{newStyle}</p>
```

Comment:
try the following, it SHOULD make it work.

```suggestion
      <style dangerouslySetInnerHTML={newStyle}></style>
```